runtime/queries/rust/highlights.scm

```diff
@@ -156,16 +156,21 @@
   "macro_rules!"
 
   "let"
-  "ref"
-  "move"
 
-  "dyn"
-  "static"
-  "const"
   "async"
 ] @keyword
 
-(mutable_specifier) @keyword.mut
```

Comment:
You removed an existing scope that some themes might target. This should be `@keyword.storage.modifier.mut`

Reply:
This is undocumented, not used in any builtin themes and I couldn't find it being used in the wild either. I'm still sure it should be removed

Reply:
I guess showing it under storage.modifier.mut is fine